test/worlds/test_base.py

```diff
@@ -19,13 +19,13 @@ def setUp(self) -> None:
         if self.auto_construct:
             self.world_setup()
 
-    def world_setup(self) -> None:
+    def world_setup(self, seed: typing.Optional[int] = None) -> None:
```

Comment:
i don't see this being used - does it mean you decided against fixing it with a set seed, but still left it in?
or is this a mistake?

Reply:
That was a utility that helped me run 15000 seeds. I left it in because I thought it might be useful for others.